common/util/goal_distance_calculator/include/goal_distance_calculator/goal_distance_calculator.hpp

```diff
@@ -47,7 +47,7 @@ class GoalDistanceCalculator
 public:
   Output update(const Input & input);
 
-  void setParam(const Param & param) { param_ = param; }
+  void setParam(const Param & param) {param_ = param;}
```

Comment:
I guess you are using uncrustify. Please use clang-format instead.

Reply:
.cpp too so maybe use pre-commit would be faster
```
 { route_ = msg_ptr; });
```

Reply:
I change to using pre-commit.
0465c0de55613cfbfaa4caf011ea51f0addb0de6